From 82d56820db4f05711b125daf46a4777e99dbdf87 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 25 Aug 2020 08:23:52 +0100
Subject: [PATCH] v2v: -o openstack: Allow guests to be converted to UEFI
 (RHBZ#1872094).

Since this output method was written the code has always been capable
of adding the hw_firmware_type = uefi image property, and this
property has been supported since at least OpenStack 12 which is years
old.

Interestingly now all of the output modes support both BIOS and UEFI.

(cherry picked from commit 5fa65a13fbbaab03cb558d0c776c17227433f1b3)
---
 v2v/output_openstack.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/v2v/output_openstack.ml b/v2v/output_openstack.ml
index 179b0edf..fdc04b02 100644
--- a/v2v/output_openstack.ml
+++ b/v2v/output_openstack.ml
@@ -390,7 +390,7 @@ object
      | None -> ""
      | Some op -> " -op " ^ op)
 
-  method supported_firmware = [ TargetBIOS ]
+  method supported_firmware = [ TargetBIOS; TargetUEFI ]
 
   (* List of Cinder volume IDs. *)
   val mutable volume_ids = []
